research/index.html

```diff
@@ -24,40 +24,47 @@
         <nav>
           <ul class="nav nav-pills float-right">
             <li class="nav-item">
-              <img src="image/quadcore_logo.png" class="nav-link active" href="#"><span class="sr-only"> ()
+              <a class="nav-link" href="#"><img src="image/quadcore_logo.png" class="nav-link active" href="#"><span class="sr-only"></a>
             </li>
             <li class="nav-item">
-              <img src="image/quadcore_logo.png" class="nav-link active" href="#"><span class="sr-only">
+              <a class="nav-link" href="#"><img src="image/user.png" class="nav-link active" href="#"><span class="sr-only"></a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="#">Contact</a>
+              <a class="nav-link" href="#"><img src="image/setting.png" class="nav-link active" href="#"><span class="sr-only"></a>
             </li>
           </ul>
         </nav>
       </div>
      </section>
 
      <section id="news">
-      <div class="jumbotron">
-        <h1 class="display-6">Processor flaw exposes 20 years of devices to new attack</h1>
-        <p class="lead">All week, the tech world has been piecing through rumors of a potentially catastrophic flaw in an entire generation of processors — but with all developers subject to a non-disclosure agreement, there were few hard facts to go on.fermentum massa justo sit amet risus.</p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/W4PO8Jygk63Xc6JIyjKqinSHqWA=/0x0:640x362/920x613/filters:focal(269x130:371x232):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195665/Screen_Shot_2011-12-16_at_11.08.27_AM.0.jpg"></p>
+      <div class="card">
+        <div class="card-body">
+          <h1 class="card-title">Processor flaw exposes 20 years of devices to new attack</h1>
+          <p class="card-subtitle mb-2 text muted">All week, the tech world has been piecing through rumors of a potentially catastrophic flaw in an entire generation of processors — but with all developers subject to a non-disclosure agreement, there were few hard facts to go on.fermentum massa justo sit amet risus.</p>
```

Comment:
Bit puzzled why we have hardcoded content here. Are these meant to be mocks before writing the frontend code?

Reply:
Yes, I intend to test.